etc/workflows/ng-retract.xml

```diff
@@ -35,7 +35,7 @@
               <label for="retractFromOaiPmh">Default OAI-PMH Repository</label>
             </li>
             <li>
-              <input id="retractFromYouTube" type="checkbox" class="configField" value="true" disabled="disabled" />
+              <input id="retractFromYouTube" type="checkbox" checked="checked" class="configField" value="true" disabled="disabled" />
```

Comment:
```
2018-04-17 11:30:58,938 | INFO  | (RetractYouTubeWorkflowOperationHandler:113) - No youtube publication found to retract in mediapackage e3df7de4-f9c0-4149-b763-2ec089f46ae7!
```

So no, it's not disabled by default but it doesn't hurt anything.  Strong feelings about changing the default?

Reply:
We can change the default if that works fine but then we should remove the `disabled="disabled"` so that people can uncheck it. Otherwise, we could just remove the option.